app/views/users/products.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= @user.name %>'s Products
+<% if logged_in? && current_user == @user %>
+  <small><%= link_to "Add a Product", new_user_product_path(@user) %></small>
+<% end %>
+</h1>
+<% if logged_in? && current_user == @user %>
```

Comment:
This line``<% if logged_in? && current_user == @user %>``  is already above, change the code so you don't repeat it